README.md

```diff
@@ -69,7 +69,6 @@ To illustrate, `bin/rails console` will run the console in the docker container
 - [Annotate](https://github.com/ctran/annotate_models) for doc the schema in the classes
 - [Better Errors](https://github.com/charliesome/better_errors) for a better error page
 - [Brakeman](https://github.com/presidentbeef/brakeman) for static analysis security
-- [Bullet](https://github.com/flyerhzm/bullet) help to kill N+1
```

Comment:
@TimoPeraza you mean this? or did I miss any other place?

Reply:
Yes, that one

Reply:
ok it's already removed